backend/TaskTree/Models/Requests/CreateTaskRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using TaskTree.Models.Responses;
+
+namespace TaskTree.Models.Requests
```

Comment:
this object represents the information passed into the controller when creating a task. when creating a task, there will only be a name, description, and weight passed in.